src/github3/repos/repo.py

```diff
@@ -259,6 +259,46 @@ def branches(self, number=-1, protected=False, etag=None):
         return self._iter(int(number), url, branch.ShortBranch, params,
                           etag=etag, headers=branch.Branch.PREVIEW_HEADERS)
 
+    @decorators.requires_app_installation_auth
+    def check_run(self, id):
+        """Return a single check run.
+
+        .. versionadded:: 1.3.0
+
+        :param int id:
+            (required), id of the check run
+        :returns:
+            the check run
+        :rtype:
+            :class:`~github3.checks.CheckRun`
+        """
+        data = None
+        if int(id) > 0:
+            url = self._build_url('check-runs', str(id),
+                                  base_url=self._api)
+            data = self._json(self._get(url), 200)
```

Comment:
We need to add the preview `Accept` header here (`check.CheckRun.CUSTOM_HEADERS`)